valayDave/metaflow
# Allow null names for artifacts

To allow artifacts without names -

Refactored Artifact type to be an array of artifacts with `name` properties instead of a dictionary.

## To test

* `cd metaflow/plugins/cards/ui`
* `yarn dev`
* Go to the URL in the output
* Look at the first row of the Artifacts table to see that it doesn't have a name

<img width="1111" alt="Screen Shot 2022-01-11 at 11 56 43 AM" src="https://user-images.githubusercontent.com/93726128/149005716-a9d649b9-3ff1-4897-a0a2-2c5f1f5d73b0.png">
